src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        for (const prop in action.extraData) {
+          state[prop] = action.extraData[prop];
+        }
```

Comment:
there is a method for that, called Object.assign, or you can use `...` spread operator. using loop ere would be bad

Reply:
used it, thanks